llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -2136,6 +2136,11 @@ def isGFX9GFX10 :
             "Subtarget->getGeneration() == AMDGPUSubtarget::GFX10">,
   AssemblerPredicate<(all_of FeatureGFX9Insts, (not FeatureGFX11Insts))>;
 
+def isGFX9GFX10GFX11 :
```

Comment:
I really hate this style of predicate. We ought to add subtarget features for everything